PULL_REQUEST_TEMPLATE

```diff
@@ -13,18 +13,12 @@ describe here. Examples: DB migration, Docker container restart/rebuild, .env ch
 
 I tested this code by (unit/integration/manual) testing&hellip;
 
-## Environment Variables
+## Extras
+- [ ] Includes a DB migration
+- [ ] PR requires other developers to make changes to their local environment after this PR has been merged: Examples: docker container restart/rebuild, .env changes
+- [ ] Includes UI Changes: add screenshots or videos
```

Comment:
This is better, but still kinda verbose IMO

Reply:
Reduced word count